plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -584,22 +586,37 @@ function OPDSBrowser:showDownloads(item)
         if acquisition.count then
             stream_buttons = {
                 {
-                    -- @translators "Stream" here refers to being able to read documents from an OPDS server without downloading them completely, on a page by page basis.
-                    text = _("Page stream") .. "\u{2B0C}", -- append LEFT RIGHT BLACK ARROW
-                    callback = function()
-                        OPDSPSE:streamPages(acquisition.href, acquisition.count, false, self.root_catalog_username, self.root_catalog_password)
-                        UIManager:close(self.download_dialog)
-                    end,
-                },
-                {
-                    -- @translators "Stream" here refers to being able to read documents from an OPDS server without downloading them completely, on a page by page basis.
-                    text = _("Stream from page") .. "\u{2B0C}", -- append LEFT RIGHT BLACK ARROW
-                    callback = function()
-                        OPDSPSE:streamPages(acquisition.href, acquisition.count, true, self.root_catalog_username, self.root_catalog_password)
-                        UIManager:close(self.download_dialog)
-                    end,
+                    {
+                        -- @translators "Stream" here refers to being able to read documents from an OPDS server without downloading them completely, on a page by page basis.
+                        text = "\u{23EE} " .. _("Page stream"),
```

Comment:
```suggestion
                        text = "\u{23EE} " .. _("Page stream"), -- Black Left-Pointing Double Triangle with Vertical Bar
```

Or something along those lines, @poire-z or @hius07 used all caps previously.